scripts/generator/graphql_generator/csharp/SDK/CartLineItems.cs.erb

```diff
@@ -213,13 +222,24 @@ namespace <%= namespace %>.SDK {
         /// cart.LineItems.AddOrUpdate(product, selectedOptions, 3);
         /// \endcode
         public void AddOrUpdate(Product product, Dictionary<string, string> selectedOptions, long? quantity = null, Dictionary<string, string> customAttributes = null) {
-            string variantId = VariantIdFromSelectedOptions(product, selectedOptions);
+            var variantId = VariantIdFromSelectedOptions(product, selectedOptions);
+            var variant = GetVariantById(product, variantId);
```

Comment:
Would it be possible to refactor this so we get `ProductVariant` from one go? Versus making `VariantIdFromSelectedOptions` and `GetVariantById` which is two for loops over `Products.variants`